services/drupal/web/modules/custom/epa_workflow/templates/epa-node-details.html.twig

```diff
@@ -0,0 +1,17 @@
+{#
+/**
+ * @file
+ *
+ * Available variables:
+ * - nid: the Node ID.
+ * - rid: Current Revision ID link.
+ * - revision_author: Latest Revision Author User Link.
```

Comment:
Please update the comment here as well to be `editor_in_chief` rather than `revision_author`